test/model.test.js

```diff
@@ -7006,79 +7006,16 @@ describe('Model', function() {
         autoCreate: false
       });
 
-      const Test = db.model('TestGH11229Var2', schema);
+      const Test = db.model('TestGH11229Var2', schema, 'TestGH11229Var2');
 
       await Test.collection.drop().catch(() => {});
       await Test.createCollection({ expires: '5 seconds' });
 
-      await Test.insertMany([
```

Comment:
Didnt i write this unit test? But i understand that on expiration test is enough and this one just checks the conversion of ms to a number.

Reply:
@Uzlopak I think that, in hindsight, testing that MongoDB actually expires the documents is a bit too brittle. The MongoDB server doesn't always expire the documents in time, and that can cause the tests to unexpectedly fail. So I think it is better to just make sure the server is configured to expire documents.